api/resource.rb

```diff
@@ -228,6 +243,9 @@ def validate
       check :properties, type: Array, item_type: Api::Type, required: true unless @exclude
       check :parameters, type: Array, item_type: Api::Type unless @exclude
 
+      check :iam_policy, type: Api::Resource::IamPolicy
+      check :exclude_resource, type: :boolean, default: false
```

Comment:
```suggestion
      check :exclude_resource, type: :boolean
```

The false default is implicit, right?

Reply:
Is it? I don't see the harm in specifying the default

Reply:
Fair enough! Yeah, there's no harm in it.